src/renderer/modals/Send/steps/StepSummary.js

```diff
@@ -68,6 +68,9 @@ export default class StepSummary extends PureComponent<StepProps> {
     const feesCurrency = getAccountCurrency(mainAccount);
     const unit = getAccountUnit(account);
     const utxoLag = txInputs ? txInputs.length >= WARN_FROM_UTXO_COUNT : null;
+    const hasNonEmptySubAccounts =
+      account.type === "Account" &&
+      some(account.subAccounts, subAccount => subAccount.balance.gt(0));
```

Comment:
same remark as on LLM.

(maybe this hasNonEmptySubAccounts  should be a helper in live-common btw)